waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/DidCreation.kt

```diff
@@ -49,9 +49,18 @@ object DidCreation {
                 }
             }
         }) {
-            getWalletService().createDid(
-                DidWebMethodName, extractDidCreateParameters(DidWebMethodName, context.request.queryParameters)
-            ).let { context.respond(it) }
+            val parameters = extractDidCreateParameters(DidWebMethodName, context.request.queryParameters)
+            runCatching {
+                getWalletService().createDid(DidWebMethodName, parameters)
+            }.onFailure {
+                if (it.message?.contains("DID already exists") == true) {
+                    context.respond(HttpStatusCode.Conflict, it.message!!)
+                } else {
+                    context.respond(HttpStatusCode.InternalServerError, it.message ?: "Internal Server Error")
```

Comment:
maybe also display the error (not just internal server error)
if so, the whole failure block could be simplified, e.g.
```kotlin
val status = if (it.message.contains("DID already exists") == true) {
    HttpStatusCode.Conflict
} else {
    HttpStatusCode.InternalServerError
}
context.respond(status, it.message?:"some-fallback-message")
```